### ISSUE TITLE
Create crawlers

### ISSUE THREAD

**Issue by ed-henrique** [ed-henrique/manga-br-search-engine#4]
Create crawlers for the following sites:

- https://www.lezhinus.com/en
- https://9anime.vc
- https://www.brmangas.net
- https://www.manganato.com
- https://www.anime-planet.com
- https://goyabu.com
- https://www.anitube.site
- https://xpanimes.com
- https://animefire.net
- https://www.crunchyroll.com
- https://animesonline.cc
- https://mangavibe.top
- https://mangatoon.mobi
- https://mangahosted.com
- https://unionleitor.top
- https://www.mangalivre.net
- https://www.novelupdates.com
- https://www.tappytoon.com/en/home/comics
- https://mangareader.to
- https://4anime.gg
- https://mangabuddy.com
- https://mangastic.me
- https://mangazim.com
- https://animesonehd.cc
- https://animesbr.biz
- https://subanimes.cc
- https://meusanimes.net
- https://goyabu.org
- https://animeyabu.com
- https://animesonlinehd.vip
- http://animesup.biz
- https://meusanimes.org
- https://animesrubro.net/lista-de-animes
- https://www.listadeanimes.com
- https://betteranime.net
- https://animefan.cc
- https://animezeira.net
- https://animesonlinen.com
- https://veanimesonline.com
- http://www.dattebane.com
- https://animexhd.com
- https://leitor.net
- https://mangadex.org
- https://mangayabu.top
- https://muitomanga.com
- https://goldenmangas.top
- https://meusmangas.net/comienzo
- https://mangaschan.com
- https://yesmangas1.com
- https://neoxscans.net
- https://mugiwarasoficial.com
- https://markscans.online
- https://mundomangakun.com.br
- https://reaperscans.com.br
- https://onepieceex.net
- https://www.leercapitulo.com
- https://mangalivre.biz
- https://www.anzmangashd.com
- https://mundowebtoon.com
- https://prismascans.net
- https://mangasoverall.com
- https://lectortmo.com
- https://taosect.com/leitor-online
- https://lightnovelreader.org
- https://www.lightnovelpub.com/hub_23091702
- https://www.readwn.com
- https://www.readlightnovel.me/hub
- https://wuxiaworld.site
- https://www.mtlnovel.com
- https://www.webnovel.com
- https://1stkissnovel.love/home-novel
- https://fullnovels.com
- https://freewebnovel.org
- https://m.webnovel.com
- https://www.lightnovelupdates.com
- https://wuxiaworldsite.com
- https://www.wuxiaworld.com
- https://allnovelfull.com
- https://novelcool.com
- https://www.foxaholic.com
- https://www.lightnovelspot.com
- https://www.novelmt.com
- https://www.novelmtl.com
- https://chrysanthemumgarden.com
- https://mangaclash.com
- https://www.webtoons.com/en
- https://www.nisakadhatu.com
- https://mangaweebs.in
- https://tapas.io
- https://1stkissmanga.io
- https://mangagg.com
- https://manhwatop.com
- https://x2manga.com

**Comment by shivam-Purohit** [comment]
hey again I solve that issue to be resolved already as well , Can I work on something . I am fine with this ?

**Comment by ed-henrique** [comment]
Sure. I'm currently creating a tutorial on how to create crawlers, and as soon as it is finished, I will link it in the README so it gets easier to contribute. I should be done with it shortly.

**Comment by ed-henrique** [comment]
> Sure. I'm currently creating a tutorial on how to create crawlers, and as soon as it is finished, I will link it in the README so it gets easier to contribute. I should be done with it shortly.

Done, please refer to [this](https://github.com/yasse-ofc/.github/blob/main/CRAWLER_CONTRIBUTING.md).

**Comment by shivam-Purohit** [comment]
Hey @ed-henrique  I have a doubt how do you identify a site as static , js rendering and all ? like do we need to browse through the site or is there any tool that we use?

**Comment by ed-henrique** [comment]
> Hey @ed-henrique I have a doubt how do you identify a site as static, js rendering, and all? like do we need to browse through the site or is there any tool that we use?

Ok, I have updated the guide because it was indeed unclear. Now you should be able to figure things out.

Answering your question, I am not aware of any tools to help with this part of the process, and it mostly comes from experience. There is a tool that helps me identify possible *tricky* sites though, called [Wappalyzer](https://www.wappalyzer.com/). It shows you the stack used in the site, so you know what technologies you are dealing with. If you ever come across *Cloudflare* on the site, it possibly has an anti-bot system, and it's a pain in the ass to bypass it.

I am thinking about creating a video showing how to create a crawler so I can go through the process in detail, but I don't know if it's necessary. Please let me know if you think this would be of any help.

TL;DR: Yes, you should browse through the site to find what crawler is best suited for it.
